src/App.js

```diff
@@ -24,18 +27,17 @@ function App() {
 
       <div className="box">
         <div className="box">
-
           <div>
             <h1 class="title">Related Items</h1>
           </div>
 
-          <ListContainer listName="Related" />
+          <ListContainer paramsId={params.id} listName="Related" />
```

Comment:
I think it's a good idea for us all to pass down the id this way, so that we don't have to destructure the params out multiple times